Frontend/ui-customer/src/Helpers/Utils.js

```diff
@@ -505,3 +505,12 @@ export const getMobileNumber = function (signatureAuthData, maskedNumber) {
     console.log("Decode mobileNumber from SignatureAuthData Error => " + err);
   }
 }
+
+export const extractKeyByRegex = (regex, text) => {
```

Comment:
you can give some generic name like `filterByRegex`

Reply:
@ashishsingh101  This name is generic only